pilot.go

```diff
@@ -134,6 +137,74 @@ func (c *chanController) SpliceOut(chanPoint *wire.OutPoint,
 	return nil, nil
 }
 
+// FeeEstimate returns the total estimated fee needed for the autopilot to open
+// channels of the given sizes. Since we currently open each channel in a
+// separate tx, the combined fee estimate will just be the sum of the
+// individual ones.
+func (c *chanController) FeeEstimate(chanSizes []btcutil.Amount) (
+	btcutil.Amount, error) {
+
+	// Find all unlocked unspent witness outputs that satisfy the minimum
+	// number of confirmations required.
+	coins, err := c.server.cc.wallet.ListUnspentWitness(
+		c.minConfs, math.MaxInt32,
+	)
+	if err != nil {
+		return 0, err
+	}
+
+	// Since we currently open each channel in a separate tx, the combined
+	// fee estimate will just be the sum of the individual ones. We attempt
+	// to mimic the coin selection done in the wallet. This won't be
+	// totally accurate (coins can be selected differently when the
+	// channels are actually being opened), but this is just an estimate
+	// soooo ¯\_(ツ)_/¯
```

Comment:
kek

Reply:
In order to make it a bit more accurate, this entire method should be nested within a call to [`WithCoinSelectLock`](https://github.com/lightningnetwork/lnd/blob/master/lnwallet/wallet.go#L1261).